### DISCUSSION TITLE
Build failed (commit 54ee34f629f91bd32c762a013ab4e102909d8cb7)

### DISCUSSION THREAD

**Discussion by uazo** [bromite/bromite#1482] [General]
@csagan5 Is it my fault or does it happen to you too?

see https://github.com/uazo/bromite-buildtools/runs/3919596647?check_suite_focus=true#step:4:49421

**Comment by csagan5** [comment]
Yes, I saw this failure and already have a local patch for it; but I see you submitted PR #1485?

**Reply by uazo** [reply]
> but I see you submitted PR #1485?

yes,  I wanted to go on with the various issues and since you and I have different times for replies :), I went ahead...
delete it "senza problemi" if you already have the solution
